docs/typescript/tutorials/queue-tutorial.md

```diff
@@ -68,6 +68,31 @@ class Tasks {
 }
 ```
 
+### Enqueue with DBOSClient
+
+[`DBOSClient`](../reference/client.md) provides a way to programmatically interact with your DBOS application from external code.
+Among other things, this allows you to enqueue workflows from outside your DBOS application.
+
+Since `DBOSClient` is designed to be used from outside your DBOS application, workflow and queue metadata must be specified explicitly.
+
+Example: 
+
+```ts
+import { DBOSClient } from "@dbos-inc/dbos-sdk";
+
+const client = await DBOSClient.create(process.env.DBOS_DATABASE_URL);
+
+type ProcessTask = typeof<Tasks.processTask>;
+await client.enqueue<Parameters<ProcessTask>>(
```

Comment:
Do we still need `Parameters<>`?

Reply:
Nope, should just be the function type. Plus, I screwed up the `typeof` statement so I will fix that too.